awslabs/eventbridge-kafka-connector
# support sending proper batches to EventBridge

Description
-----------

Sending proper batches to EventBridge w.r.t. 256kb size limit. A single batch of an item > 256kb is allowed, see discussion on #35.

Test Steps
-----------

```
$ mvn clean test
```

Checklist:
----------




Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
